// RESULT: FAIL
def main() {

	return 3 < 4 < 5;
}

